upgrade.js

```diff
@@ -15,7 +17,28 @@ function howToInstallElmFormat(){
 
 function displayHintForNonUpgradedPackage(packageName){
     process.stdout.write(`WARNING: ${packageName} has not been upgraded to 0.18 yet!\n`);
-    process.stdout.write(`WARNING: You can make an issue for 0.18 upgrade here: https://github.com/${packageName}\n`)
+    packagesRequiringUpgrade.push(packageName);
+}
+
+function displaySuccessMessage() {
+  process.stdout.write(
+    'SUCCESS! Your project\'s dependencies and code have been upgraded.\n'
+    + 'However, your project may not yet compile due to API changes in your\n'
+    + 'dependencies.\n\n'
+    + 'See https://github.com/elm-lang/elm-platform/blob/master/upgrade-docs/0.18.md\n'
+    + 'and the documentation for your dependencies for more information.\n\n'
+  );
+
+  if (packagesRequiringUpgrade.length > 0) {
+    process.stdout.write(
+      'WARNING! Several of your dependencies have not yet been upgraded to \n'
+      + 'support Elm 0.18. You can create an issue to request the packages be \n'
+      + 'upgraded here:\n');
+    packagesRequiringUpgrade.forEach(function(packageName) {
+      process.stdout.write('  - https://github.com/' + packageName + '/issues\n');
+    });
+    process.stdout.write('\n');
```

Comment:
oh making the PRs...   Hmm.. That's a cool idea!  I think if we wanted to automate the PRs, it would be better to separately just go through all the published packages and create the PRs (as an independent, one-time script), and not have elm-upgrade to that, since so many people will be running it.

Also, I didn't want to automate issue creation because most likely there will already be an issue created, and I don't want all package maintainers to get a bunch of duplicate issue reports.